src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/MemberInfoExtensions.cs

```diff
@@ -2,16 +2,18 @@
 using System.Linq;
 using System.Reflection;
 using Microsoft.AspNetCore.Mvc;
-
+using System;
 namespace Swashbuckle.AspNetCore.SwaggerGen
 {
     public static class MemberInfoExtensions
     {
         public static IEnumerable<object> GetInlineOrMetadataTypeAttributes(this MemberInfo memberInfo)
         {
+            if (memberInfo==null) throw new NullReferenceException(nameof(memberInfo));
```

Comment:
As a general rule, I prefer to put the responsibility of null checking on the extension method caller rather than inside the method itself. This is the way it is for all other instance methods and I prefer to be consistent in the case of extension methods aswell.

Reply:
So instead I should return empty enumerable in order to be consistent?